dataikuapi/dss/modelevaluationstore.py

```diff
@@ -0,0 +1,298 @@
+        }
+        res = self.client._perform_json("POST", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id), 
+                                            body = obj)
+        run_id = res['id']
+        return DSSModelEvaluation(self, run_id)
+
+
+class DSSModelEvaluationStoreSettings:
+    """
+    A handle on the settings of a model evaluation store
+
+    Do not create this class directly, instead use :meth:`dataikuapi.dss.DSSModelEvaluationStore.get_settings`
+    """
+
+    def __init__(self, model_evaluation_store, settings):
+        self.model_evaluation_store = model_evaluation_store
+        self.settings = settings
+
+    def get_raw(self):
+        return self.settings
+
+    def save(self):
+        self.model_evaluation_store.client._perform_empty(
+                "PUT", "/projects/%s/modelevaluationstores/%s" % (self.model_evaluation_store.project_key, self.model_evaluation_store.mes_id),
+                body=self.settings)
+
+class DSSModelEvaluation:
+    """
+    A handle on a model evaluation
+
+    Do not create this class directly, instead use :meth:`dataikuapi.dss.DSSModelEvaluationStore.get_model_evaluation`
+    """
+
+    def __init__(self, model_evaluation_store, run_id):
+        self.model_evaluation_store = model_evaluation_store
+        self.client = model_evaluation_store.client
+        # unpack some fields
+        self.run_id = run_id
+        self.project_key = model_evaluation_store.project_key
+        self.mes_id = model_evaluation_store.mes_id
+
+    def get_full_info(self):
+        """
+        Retrieve the model evaluation with its performance data
+        """
+        return self.client._perform_json(
+                "GET", "/projects/%s/modelevaluationstores/%s/runs/%s" % (self.project_key, self.mes_id, self.run_id))
+
+    def get_settings(self):
+        """
+        Set the definition of this model evaluation
+        """
+        data = self.client._perform_json(
+                "GET", "/projects/%s/modelevaluationstores/%s/runs/%s/settings" % (self.project_key, self.mes_id, self.run_id))
+        return DSSModelEvaluationSettings(self, data)
+
+    def delete(self):
+        """
+        Remove this model evaluation
+        """
+        obj = [self.run_id]
+        self.client._perform_json(
+                "DELETE", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id), body=obj)
+
+    ########################################################
+    # Model evaluation contents
+    ########################################################
+    
+    def list_contents(self):
+        """
+        Get the list of files in the model evaluation
+        
+        Returns:
+            the list of files, as a JSON object
+        """
+        return self.client._perform_json(
+                "GET", "/projects/%s/modelevaluationstores/%s/runs/%s/contents" % (self.project_key, self.mes_id, self.run_id))
+
+    def get_file(self, path):
```

Comment:
Note for later: this seems pretty low level, do we really want to expose raw files ? There is so much way things can go wrong